part-i.md

```diff
@@ -194,7 +194,7 @@ fireTheMissiles
 {% tabs %}
 {% tab title="TypeScript" %}
 ```typescript
-const withInner = (toLog: string): () => {
+const withInner = (toLog: string): void => {
```

Comment:
There is no unit type in TypeScript. This function can either explicitly return `void` or the return type can be skipped (and inferred from the body).